src/vm/excep.cpp

```diff
@@ -7233,8 +7233,9 @@ EXTERN_C void JIT_WriteBarrier_Debug_End();
 EXTERN_C void FCallMemcpy_End();
 #endif
 
-static
-bool IsIPExcluded(UINT_PTR uControlPc)
+// Check if the passed in instruction pointer is in one of the
+// JIT helper functions.
+bool IsIPInJitHelper(UINT_PTR uControlPc)
```

Comment:
JIT helpers are all methods in vm\jithelpers.cpp ... maybe a better name would be ShouldHandleJitHelperFault ?